src/Analysis/Ast/Impl/Modules/Resolution/MainModuleResolution.cs

```diff
@@ -135,7 +135,7 @@ public MainModuleResolution(string root, IServiceContainer services)
 
         private async Task<IReadOnlyList<string>> GetInterpreterSearchPathsAsync(CancellationToken cancellationToken = default) {
             if (!_fs.FileExists(Configuration.InterpreterPath)) {
-                _log?.Log(TraceEventType.Error, "Interpreter does not exist:", Configuration.InterpreterPath);
+                _log?.Log(TraceEventType.Warning, "Interpreter does not exist:", Configuration.InterpreterPath);
```

Comment:
We should pop up error saying that LS can't continue. I.e. the `IUIService.ShowMessageAsync` thing.

Reply:
![image](https://user-images.githubusercontent.com/5341706/54784422-1abf9a00-4be1-11e9-9dec-ce349b6e65ad.png)

This alright? (For the exception, as an example.)

Reply:
Only nit about my wording is "getting" feels weird.

Reply:
IMO explicit message that interpreter is missing is more straightforward. Also, it should be an error as LS is not functional without it.

Reply:
OK, you probably missed my comment that it was an example of the caught exception; this is what you'd get if it were the interpreter missing:

![image](https://user-images.githubusercontent.com/5341706/54785950-c8807800-4be4-11e9-99d3-747d39337627.png)

Reply:
I've seen it but it is a warning. Its not like "may not be available" :-) it definitely won't it and most probably we'll just crash.